### ISSUE TITLE
Deleting a container leaves leftover files

### ISSUE THREAD

**Issue by poisa** [kitematic/kitematic#246]
I created two containers "hello-world-nginx" and the official mongoDb one. After deleting them, the folder /Users/<my user>/Kinematic, still had all the files referenced by these containers.

**Comment by jeffdm** [comment]
Thanks for creating an issue! We keep them now in case you want to re-use the files, but I understand how that can leave a bunch of "leftovers". Would it make sense to delete the files if the container is explicitly deleted?

**Comment by poisa** [comment]
>Would it make sense to delete the files if the container is explicitly deleted?

Yes, I think so. Or maybe, in the dialog box that prompts you to confirm the deletion, add one button to delete the container and another to delete the container + local files (or a checkbox). I'd prefer two buttons because it's easier to do with hotkeys. 

Also, it would be great if the the path would be shown somewhere. Something like breadcrumbs that you can double-click to open the location in a Finder window (or if you want to follow Apple's idioms: Right click > Show in Finder)

![screen shot 2015-03-12 at 10 55 03 pm](https://cloud.githubusercontent.com/assets/348248/6632327/d73ba43e-c90a-11e4-8bf9-9218263295c1.png)

Just a thought...

**Comment by jeffdm** [comment]
Ah - that's a good idea. Thanks for the input on how to make this better.
